### PR TITLE
https://issues.redhat.com/browse/ACM-8506 OCP 3.11 unsupported

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8506

### CODE REVIEW

**Review by oafischer — APPROVED** [review]
lgtm

File: release_notes/known_issues_continuity.adoc [hunk 1]
```diff
@@ -30,6 +30,14 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
 [#known-issues-backup-restore]
 == Backup and restore known issues
 
+[#restore-ocp-not-supported]
+=== Restoring {ocp-short} 3.11 managed cluster is not supported
+//2.9:ACM-8472
+
+If you have {ocp-short} 3.11 managed clusters and you restore the cluster, the restored data is not connected to the new hub cluster automatically.
```
Comment (birsanv) [inline release_notes/known_issues_continuity.adoc]: Change this

If you have {ocp-short} 3.11 managed clusters and you restore the cluster, the restored data is not connected to the new hub cluster automatically.

to 

If you have {ocp-short} 3.11 managed clusters and you restore the cluster, the managed cluster is not connected to the new hub cluster automatically.

**Review by birsanv — CHANGES_REQUESTED** [review]

File: release_notes/known_issues_continuity.adoc (hunk 1 above)
Comment (dockerymick) [inline release_notes/known_issues_continuity.adoc]: @birsanv thanks ! Adding the suggestion to make the change 

```suggestion
If you have {ocp-short} 3.11 managed clusters and you restore the cluster, the managed cluster is not connected to the new hub cluster automatically. 
```

Comment (dockerymick) [reply release_notes/known_issues_continuity.adoc]: Updated

**Review by birsanv — APPROVED** [review]
lgtm

**Pull request merged**